sklearn/metrics/tests/test_ranking.py

```diff
@@ -494,10 +534,15 @@ def test_precision_recall_curve_pos_label():
 def _test_precision_recall_curve(y_true, probas_pred):
     # Test Precision-Recall and aread under PR curve
     p, r, thresholds = precision_recall_curve(y_true, probas_pred)
-    precision_recall_auc = auc(r, p)
+    precision_recall_auc = _average_precision_slow(y_true, probas_pred)
+    interpolated_average_precision = _interpolated_average_precision_slow(
+        y_true, probas_pred)
     assert_array_almost_equal(precision_recall_auc, 0.85, 2)
     assert_array_almost_equal(precision_recall_auc,
                               average_precision_score(y_true, probas_pred))
+    assert_equal(interpolated_average_precision,
+                 average_precision_score(y_true, probas_pred,
+                                         interpolation='eleven_point'))
     assert_almost_equal(_average_precision(y_true, probas_pred),
```

Comment:
The other checks are still failing with a lower tolerance on `assert_almost_equal`. For some reason, `average_precision_score` is coming out as 0.85899092682932521 in travis. On my machine, I get 0.85873118656958514, which is equal to 12 decimal places with the 'slow' implementation in `test_rankings.py`. I can't recreate the 0.85899... number. Any ideas? The difference seems too big to be a 32 vs 64 bit problem, but we are adding up lots of decimals (that are themselves products of decimals...)